kind/cmd/kind/main.go

```diff
@@ -14,19 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This package is a stub main wrapping cmd.Run()
+// This package is a stub main wrapping root.Main()
 package main
 
 import (
-	"fmt"
-	"os"
-
-	"k8s.io/test-infra/kind/cmd/kind/cmd"
+	"k8s.io/test-infra/kind/cmd/kind/root"
 )
 
 func main() {
-	if err := cmd.Run(); err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	root.Main()
```

Comment:
Is this file worth keeping around? AFAICT it is not referenced, and is just provided as an alternative to `go install k8s.io/test-infra/kind`?

May be best to remove to prevent confusion 😄

Reply:
+ if this were removed, the exported `Main` function could be removed from `cmd/kind/root` (and instead we could include the os.Exit in `k8s.io/test-infra/kind.main` only

Reply:
Yeah I was on the fence about this as still "ok kind/cmd/kind" is a binary, it's just also exported at kind/. I originally was going to use a symlink but go doesn't seem to support this and symlinks make for build fun in general.

If we move Main() up to here we have to drop cmd/kind being a useable binary path. That seems fine to me though. I'd also think moving the root Command up to here would make it more obvious to anyone not familiar with that pattern.